scripts/deploy_testing_secret.sh

```diff
@@ -16,8 +16,8 @@ cat <<EOF | $k apply -f -
 apiVersion: v1
 kind: Secret
 metadata:
-  name: k8sinfra-controller-settings
-  namespace: k8s-infra-system
+  name: aso-controller-settings
```

Comment:
I'm not sure if the format is exactly the same or not but ideally we would use the same secret + format as the old controller I think?

It's called: `azureoperatorsettings`.

I guess maybe there's an argument for using a separate secret and just having the Helm chart populate them both (so that we can deprecate fields we don't want/like in the existing secret?)

Reply:
I'm wary of mixing secrets from both versions since we probably want to change this format...

Reply:
I'll merge this now and we can address what to do about combined settings later.